app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CategoriesController < ApplicatinController
+    before_action :set_task, only: [:update, :destroy]
```

Comment:
__[rubocop]__
* Style/IndentationWidth: Use 2 (not 4) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)